my-sms/db/schema.rb

```diff
@@ -23,6 +22,13 @@
     t.string   "gender"
     t.datetime "created_at",  :null => false
     t.datetime "updated_at",  :null => false
+    t.integer  "title_id"
```

Comment:
consider using `t.references`

Reply:
Ignore this, I'm looking at the wrong file 😅

Reply:
I use this in the migration code, should I be editing the schema.rb too?